ebusd-2.1.x/de/vaillant/c0.wtw04.csv

```diff
@@ -0,0 +1,75 @@
+# type (r[1-9] w u ),circuit,name,[comment],[QQ],ZZ,PBSB,[ID],field1,part (m/s),datatypes/templates,divider/values,unit,comment
+#,recov,RecoVair 275,,,,,,,,,,,
+############ Luftvolumen################,,,,,,,,,,,,,
+*r,,,,,,B509,0D,,,,,,
+*w,,,,,,B509,0E,,,,,,
+w,,Lüftung = 50m³/h,Lüftung = 50m³/h,,,,1500a600,,,,,,
```

Comment:
is this a linear correlation between the qm/h and the value as UIN? if so, make it a field instead of defining tons of messages. same applies to the other translation I guess